client/packages/common/src/ui/layout/tables/material-react-table/useMaterialTableColumns.tsx

```diff
@@ -37,7 +37,6 @@ export const useMaterialTableColumns = <T extends MRT_RowData>(
                   alignment === 'right'
                     ? {
                         justifyContent: 'flex-end',
-                        paddingRight: '2em', // Padding to account for header icons
```

Comment:
This is not necessary now that we remove the "three-dot" icon from the headers last week :)